### PR TITLE
add assert hints tests

### PR DESCRIPTION
**Pull request by SantiagoPittella**
TESTING STATUS:

* pow.cairo: FAILING (Issue with ids structs)
    * POW 
* is_nn.cairo: OK
    * IS_NN
    * IS_NN_OUT_OF_RANGE
* is_positve.cairo: OK
    * IS_POSITIVE
* assert_not_zero.cairo: OK
    * ASSERT_NOT_ZERO
* assert_le_felt.cairo: OK
    * ASSERT_LE_FELT 
* assert_lt_felt.cairo: OK
    * ASSERT_LT_FELT
* assert_not_equal.cairo: OK
    * ASSERT_NOT_EQUAL

Extra: 
* Remove `is_nn_hint.cairo`, (it was a duplicate of i`s_nn.cairo`)
* delete unnecessary prints from [assert_nn_hint.cairo](https://github.com/lambdaclass/cairo-rs-py/pull/56/files#diff-71103e2d325bed1ec97553345299ec96f23adeda9f8a39ef4c963181ffae0835)

### CODE REVIEW

File: cairo_programs/pow_hint.cairo [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    assert_le(n_steps, 251)
+    return (res=locs.res)
+end
+
+func main{range_check_ptr}():
+    let a : felt = 12
+
+    let a_pow = pow(a, 2)
+
+    %{
```
Comment (Oppen) [inline cairo_programs/pow_hint.cairo:63]: Should this have a `# TEST` line to make sure it runs on the Python hint executor?

Comment (pefontana) [reply cairo_programs/pow_hint.cairo]: yes, Done!

**Comment by pefontana** [comment]
https://trello.com/c/wrfklXf1/24-test-common-library-hints-asserts

**Review by Juan-M-V — APPROVED** [review]

File: hints_tests.py [hunk 2]
```diff
@@ -12,11 +12,16 @@ def test_program(program_name: str):
     test_program("hint_print_vars")
     test_program("vm_scope_hints")
     test_program("is_le_felt_hint")
-    test_program("assert_le_felt_hint")
-    test_program("is_nn_hint")
     test_program("ec_mul_inner")
-    # test_program("ec_negate")
+    # test_program("ec_negate") # FAILING (Issue with ids structs)
```
Comment (Oppen) [inline hints_tests.py:16]: This should now be working.

Comment (pefontana) [reply hints_tests.py]: Yes, now is failling due to:
`ValueError: Variable value not present in current execution scope`

**Pull request merged**